evap/student/templates/student_vote.html

```diff
@@ -173,7 +176,7 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
 
         <script type="module">
             import {initTextAnswerWarnings} from "{% static 'js/text-answer-warnings.js' %}";
-
+            const lastSavedStorageKey = "student-vote-{{evaluation.id}}-{{request.user.id}}"
```

Comment:
```suggestion
            const lastSavedStorageKey = "student-vote-last-saved-at-{{evaluation.id}}-{{request.user.id}}";
```
:paintbrush: Just to avoid possible collisions if we want to add more local storage items for this page